src/main/java/com/google/step/finscholar/servlets/CollegeServlet.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.servlets;
+
+import com.google.gson.Gson;
+import com.google.step.finscholar.data.College;
+import com.google.step.finscholar.data.CollegeData;
+import com.google.step.finscholar.servlets.JsonConverter;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet that returns colleges and their associated data. */
+@WebServlet("/college-data")
+public class CollegeServlet extends HttpServlet {
+
+  public static final String JSON_CONTENT_TYPE = "application/json;";
```

Comment:
I think we'll need another class to hold all string constants. (This is awkward.. I'm writing scholarshipServlet now) I will write one on my branch and we can coordinate on this tomorrow.

Reply:
I agree!